requre/helpers/files.py

```diff
@@ -123,34 +123,30 @@ def guess_args(cls, func: Callable) -> Any:
 
         @functools.wraps(func)
         def store_files_int(*args, **kwargs):
+            def int_dec_fn(pathname_arg, keys_arg):
+                if isinstance(pathname_arg, str):
+                    if STORAGE.is_write_mode:
+                        if os.path.exists(pathname_arg):
+                            cls._copy_logic(
+                                STORAGE, pathname=pathname_arg, keys=keys_arg
+                            )
+                    else:
+                        try:
+                            cls._copy_logic(
+                                STORAGE, pathname=pathname_arg, keys=keys_arg
+                            )
+                        except PersistentStorageException:
+                            pass
```

Comment:
What about this:
```python
def int_dec_fn(pathname_arg, keys_arg):
    if not isinstance(pathname_arg, str):
        return

    if STORAGE.is_write_mode and os.path.exists(pathname_arg):
        cls._copy_logic(STORAGE, pathname=pathname_arg, keys=keys_arg)
    elif not STORAGE.is_write_mode:
        try:
            cls._copy_logic(STORAGE, pathname=pathname_arg, keys=keys_arg)
        except PersistentStorageException:
            pass
```

(I don't like going so deep.)

Reply:
okay